microsetta_admin/server.py

```diff
@@ -578,14 +578,28 @@ def _check_sample_status(extended_barcode_info):
     return warning
 
 
+def group_observations(scans_info):
```

Comment:
Is this function still necessary with the API only returning one row per scan?

Reply:
Ah yeah, it isn't. Removing.